lib/fetch/body.js

```diff
@@ -151,22 +151,18 @@ function extractBody (object, keepalive = false) {
         if (typeof value === 'string') {
           bodyLength +=
             prefixLength +
-            Buffer.byteLength(`; name="${escape(normalizeLinefeeds(name))}"`) +
-            Buffer.byteLength(`\r\n\r\n${normalizeLinefeeds(value)}\r\n`)
+            Buffer.byteLength(`; name="${escape(normalizeLinefeeds(name))}"\r\n\r\n${normalizeLinefeeds(value)}\r\n`)
         } else {
           bodyLength +=
             prefixLength +
-            Buffer.byteLength(`; name="${escape(normalizeLinefeeds(name))}"`) +
-            (value.name ? Buffer.byteLength(`; filename="${escape(value.name)}"`) : 0) +
+            Buffer.byteLength(`; name="${escape(normalizeLinefeeds(name))}"` + value ? `; filename="${escape(value.name)}"` : '') +
```

Comment:
```suggestion
            Buffer.byteLength(`; name="${escape(normalizeLinefeeds(name))}"` + value.name ? `; filename="${escape(value.name)}"` : '') +
```

To match the current behavior.